userDetails.py

```
@@ -0,0 +1,18 @@
+
+# Create and print welcome message
+message = f"""
+Welcome, {full_name}!
+
+Your username is {username}, and it's great to have someone from {country} here.
+We see that you prefer communicating in {language}, which is awesome.
+
+We’re glad to have you onboard—enjoy your experience!
+"""
+
+priint(message)
```

Ruff (0.15.6)

[error] Undefined name `priint`

(F821)